### DISCUSSION TITLE
Which package will shipped to user

### DISCUSSION THREAD

**Discussion by musta20** [vercel/next.js#35928] [Help] [answered]
According to next js docs when building server side rendering component  it's better just to type the logic of getting the data from the db instead of fetching it because you are in the server fetching the data is waste of resource that mean iam gone use orm like serialize my question is  will orm package i install shipped to the user .

 
```
import userModle from '../../../model/userModle';
 
export async function getStaticProps({ params }) {
 
   const user = await userModle.User.findOne({ where: { username: params.username } });
    return {
     props: {
       id: user.dataValues.id,
       Title: user.dataValues.Title,
      }
   }
  }
```

**Comment by icyJoseph** [comment, marked as answer]
Next.js tries to figure out which code is used only by the server side set of functions, and removes that code from the client side bundle. You can test this out yourself with this tool: https://next-code-elimination.vercel.app/.

I'd also recommend using https://www.npmjs.com/package/@next/bundle-analyzer, to get a visualisation of which bundles are being packaged.
